hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -10,6 +10,8 @@ parameters:
   required: true
 - name: CHANNEL
   value: staging
+- name: FEDRAMP
+  required: true
```

Comment:
i recommend not making this required, and set the default value to be `"false"`.